README.md

```diff
@@ -5,39 +5,278 @@
 
 This is a set of tools and APIs for issuers of zk-proof credentials, designed to be extensible. It allows an authenticated user to create schemas for issuing and managing credentials of identities. It also provides a web-based [frontend (UI)](ui/README.md) to manage issuer schemas, credentials and connections.
 
-## Installation
+This repository is for anyone to create their own [issuer node](https://0xpolygonid.github.io/tutorials/issuer-node/issuer-node-overview/) for Polygon ID.
 
-There are two options for installing and running the server alongside the UI.
+---
 
-### Option 1 - Using Docker only
+## Installation
 
-Running the app with Docker allows for minimal installation and a quick setup. This is recommended **for evaluation use-cases only**, such as local development builds.
+There are two options for installing and running the server alongside the UI:
 
-#### Requirements for Docker-only
+1. [Docker Quick Start Guide](#docker-quick-start-guide)
+2. [Standalone Mode Guide](#standalone-mode-guide)
 
-- [Docker Engine](https://docs.docker.com/engine/) 1.27+
-- Makefile toolchain
-- Unix-based operating system (e.g. Debian, Arch, Mac OS X)
+### Docker Quick Start Guide
 
-_NB: There is no compatibility with Windows environments at this time._
+Running the app with Docker allows for minimal installation and a quick setup. This is recommended **for evaluation use-cases only**, such as local development builds.
 
-#### Setup for Docker-only
+**TL;DR Steps:**
```

Comment:
This isn't TL;DR either. It's literally the quick start steps, as indicated by the header. I would remove this, but if this must be a header, I would make it one (`####`).

Reply:
I might recommend that think about removing this in another version because although it does give a high-level overview, it doesn't follow the complete steps to understand how things are working. 

That or we look at revising it with all the make commands for an even quicker overview of all the commands that need to be run.

Reply:
Then I would replace this with `#### Quick start setup steps`.